spec/features/accessibility/idv_pages_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 require 'axe/rspec'
 
-feature 'Accessibility on IDV pages', :js do
+feature 'Accessibility on IDV pages', :js, :accessibility do
```

Comment:
I suppose you don't need tags for that. You can just do `rspec spec/features/accessibility`